docs/source/prereqs.md

```diff
@@ -115,6 +115,57 @@ Optional: Install the latest version of [Go](https://golang.org/doc/install) (on
 
 Optional: Install the latest version of [jq](https://stedolan.github.io/jq/download/) (only required for the tutorials related to channel configuration transactions).
 
+### Distros with SELinux enabled
```

Comment:
I'm ok with having this content in a test-network specific doc topic, or in fabric-samples repository itself. But I'd rather not clutter the main project prereq topic for information that is specific to a single sample.

Reply:
Happy to help!

You're right. I have two proposals then:
1. How about [here](https://twogiants-fabric.readthedocs.io/en/selinux-test-network-fix/test_network.html#troubleshooting) right above _Troubleshooting_ in the main documentation about the test network? With a mention of this case in _Before you begin_ up top [here](https://twogiants-fabric.readthedocs.io/en/selinux-test-network-fix/test_network.html#before-you-begin).
2. Or [here](https://github.com/hyperledger/fabric-samples/tree/main/test-network#podman) in the fabric-samples repo test network `README.md` below _Podman_. And then a reference to this entry in _Before you begin_ [here](https://twogiants-fabric.readthedocs.io/en/selinux-test-network-fix/test_network.html#before-you-begin) as in proposal 1.

Reply:
I like your proposal 1. Add the content to the test network troubleshooting section and mention/link to it closer to the top. Thanks!

Reply:
Added changes as discussed.